### PR TITLE
fix: fixed inconsistent border Radius and Icon for theme

### PR DESCRIPTION
**Pull request by Rahul-ku-Mo**
Fixed the following :
1. Border radius slider not working correctly in theme settings
2. Change the icon of theme settings to one same as topbar
3. Handle dark mode theming based on canvas dark mode instead of switch for colors

### CODE REVIEW

File: src/core/components/sidepanels/panels/theme-configuration/BorderRadiusInput.tsx [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+
   return (
     <input
       type="range"
       min="0"
       step="1"
       max="30"
       disabled={disabled}
-      defaultValue="15"
+      defaultValue={value.replace("px", "")}
```
Comment (surajair) [inline src/core/components/sidepanels/panels/theme-configuration/BorderRadiusInput.tsx:19]: Will is save as `px` at the time of save?

File: src/core/components/sidepanels/panels/theme-configuration/ThemeConfigPanel.tsx [hunk 2]
```diff
@@ -178,7 +179,11 @@ const ThemeConfigPanel: React.FC<ThemeConfigProps> = React.memo(({ className = "
           <div className="py-3">
             <h4 className="text-sm font-bold">{t("Border Radius")}</h4>
             <div className="flex items-center gap-4">
-              <BorderRadiusInput disabled={selectedPreset === "preset"} onChange={handleBorderRadiusChange} />
+              <BorderRadiusInput
+                value={themeValues.borderRadius}
+                disabled={selectedPreset === "preset"}
```
Comment (surajair) [inline src/core/components/sidepanels/panels/theme-configuration/ThemeConfigPanel.tsx]: why is this disabled for preset?

**Review by surajair — CHANGES_REQUESTED** [review]

**Review by surajair — APPROVED** [review]

File: src/core/components/sidepanels/panels/theme-configuration/ThemeConfigPanel.tsx (hunk 2 above)
Comment (Rahul-ku-Mo) [reply src/core/components/sidepanels/panels/theme-configuration/ThemeConfigPanel.tsx]: This logic we don't anymore. It's there from the previous preset usage.

File: src/core/components/sidepanels/panels/theme-configuration/BorderRadiusInput.tsx (hunk 1 above)
Comment (Rahul-ku-Mo) [reply src/core/components/sidepanels/panels/theme-configuration/BorderRadiusInput.tsx:19]: We only need this for once . Yeah it will save as px

**Pull request merged**